standalone-metastore/packaging/src/assembly/bin.xml

```diff
@@ -29,7 +29,7 @@
     <format>tar.gz</format>
   </formats>
 
-  <baseDirectory>apache-hive-metastore-${project.version}-bin</baseDirectory>
+  <baseDirectory>apache-${project.parent.artifactId}-server-${project.version}-bin</baseDirectory>
```

Comment:
why change?

Reply:
renamed it to `${project.parent.artifactId}-${project.version}-bin` to keep the same pattern with the old releases

Reply:
@dengzhhu653 fyi, that is not an archive name, but the folder inside

Reply:
now the dir is `apache-hive-metastore-4.2.0-SNAPSHOT-bin` after unzipping the tarball